packages/react-instantsearch/src/core/createInstantSearch.js

```diff
@@ -41,6 +41,7 @@ export default function createInstantSearch(defaultAlgoliaClient, root) {
     render() {
       return (
         <InstantSearch
+          {...this.props}
```

Comment:
I don't think we want to go back to this spread way of doing things. Best would be to be explicit about all the parameters that will be passed down. It seems that we're missing two things: 
 - `searchParameters`
 - `createURL`